aleph-client/Cargo.toml

```diff
@@ -14,6 +14,10 @@ features = ["staking-xt"]
 sp-core = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", features = ["full_crypto"] }
 sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.13", default-features = false }
 pallet-staking = {git = "https://github.com/paritytech/substrate.git",branch = "polkadot-v0.9.13", default-features = false }
+pallet-treasury = {git = "https://github.com/paritytech/substrate.git",branch = "polkadot-v0.9.13", default-features = false }
+pallet-aleph = {path = "../pallets/aleph", default-features = false}
+primitives = { path = "../primitives", default-features = false }
+aleph-runtime = { path = "../bin/runtime"}
```

Comment:
I remember @fbielejec  raising a point we should be not doing that, as it increases the compilation time of `e2e` crate. Personally, I was okay with that, technically e2e are those which tests the runtime, but maybe @fbielejec wants to make similar point here? The code of `e2e` tests can be simplified (e.g. we don't need to create our own `Header` or `Block` types, we can just re-use types runtime creates etc)

Reply:
Ah now seen it is `aleph-client` crate, not `e2e`. I think `aleph-client` should not depend on the runtime as it should be a lightweight library wrapping over the `substrate-api-client` ? Maybe we can move this dep to `e2e` create?
CC: @timorl

Reply:
I am using it here only because of one type definition I think. Maybe it would be a good idea to extract such things from the runtime into primitives? Then depending on primitive is not such a burden as depending on the runtime, wdyt?

Reply:
What type is it? Ideally, we'd like to keep `aleph-client` lightweight, (in terms of compile-time for example). Seem we have two options
* we can move `aleph-runtime` dep to `e2e` crate as it is where it is used,
* we can add this one type which is required to `aleph-client`
Is any of them okay for u?